### DISCUSSION TITLE
Invalid `prisma.collection.findMany()` invocation

### DISCUSSION THREAD

**Discussion by johnchonn** [prisma/prisma#23858] [Q&A]
### Question

I don't know why but periodically, my app will just crash with an error:

```
Error: 
Invalid `prisma.collection.findMany()` invocation:


Error occurred during query execution:
ConnectorError(ConnectorError { user_facing_error: None, kind: QueryError(PostgresError { code: "42P05", message: "prepared statement \"s0\" already exists", severity: "ERROR", detail: None, column: None, hint: None }), transient: false })
```

And the error it claims to be coming from is this:
```
async function getData() {
  const clothingData = await prisma.collection.findMany({
    relationLoadStrategy: 'join',
    include: {
      sizes: true,
    },
  });
  ```
  
  I also do have this PreviewFeatures in my schema:
  ```
  generator client {
  provider = "prisma-client-js"
  previewFeatures = ["relationJoins"]
}
```
  
  This doesn't happen every time. Sometimes, for days, I can go without any problems and randomly, my app will crash and show me this error. I have been stuck on this for quite some time and I just cannot seem to find why this is happening. Any sort of help would be much appreciated!! 

### Information about Prisma Schema, Client Queries and Environment (optional)

```prisma
// Add your schema.prisma
```

```ts
// Add any relevant Prisma Client queries here
```

- OS: 
- Database: 
- Node.js version: 

- Run `prisma -v` to see your Prisma version and paste it

**Comment by ludralph** [comment]
Hi @johnchonn 👋 

Thank you for raising this question.

Are you using  `pgbouncer`? If yes, then you will need to follow the workaround detailed in this section of the [documentation](https://www.prisma.io/docs/orm/prisma-client/setup-and-configuration/databases-connections/pgbouncer#prisma-migrate-and-pgbouncer-workaround).

**Reply by johnchonn** [reply]
Hello! No I am not using `pgbouncer` ...

**Reply by ludralph** [reply]
Hi @johnchonn 

Do you still experience this issue? What database are you using and where are you deploying your app?

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!;
